app/styles/_action-chip.less

```diff
@@ -0,0 +1,39 @@
+.action-chip {
+  margin: 0 5px 2px 0;
+  font-size: 12px;
+  display: flex;
+  flex-direction: row;
+  .item {
+    padding: .2em .6em .3em;
+    // explicit radius as is possible 1 node will receive all rules
+    &:first-child {
+      border-top-left-radius: 2px;
+      border-bottom-left-radius: 2px;
+    }
+    &:last-child {
+      border-top-right-radius: 2px;
+      border-bottom-right-radius: 2px;
+    }
+    &:nth-child(3) {
```

Comment:
I can post another screenshot if you like. It works like this:
```
key|value| x
```
There is only possibly a third segment, if you use both key and value and action icon, and in that case the third gets a line to the left.  I'm only using `key|x` but tried to make this generic.  Its based on the `k8s-label` loosely, but this isnt a kubernetes label, so it seemed that the original wasn't generic enough.